OPERATING.md

```diff
@@ -0,0 +1,79 @@
+  - [As a Cooperative](#as-a-cooperative-2)
+  - [As a Person](#as-a-person-2)
+  - [As a Classic Non-Profit, Government, or Corporation](#as-a-classic-non-profit-government-or-corporation)
+
+
+Convene is as much a socioeconomic project as it is a technological one. It serves as a platform for designing, building and hosting web software applications that grant super-powers to mutual-aid and mutual benefit solidarity economy organizations.
+
+Convene is deployed as a `Neighborhood` on commodity cloud infrastructure. It allows an `Operator`, (either a person or a worker and community-owned mutual-benefit technology cooperative) to support the digital operations of mutual-aid and mutual-benefit organizations (or `Neighbor`s).
+
+Where `Contributors` labor to research, design, test, and build software that delights `Neighbor`s and grants them superpowers, `Operators` labor to distribute this software to `People`. `Operators`:
+
+- Nurture the `Neighborhood` by setting and maintaining healthy boundaries between `People`, `Neighbor`s and other `Neighborhood`s.
+- Welcome `People` to the `Neighborhood` and support them in becoming a good `Neighbor`.
+- Help `Neighbor`s get the most from their `Neighborhood` through carework, gluework and technical support.
+
+This guide covers the social, economic, and technological aspects of operating a Convene `Neighborhood`. Each section starts by describing the shared context, and then splitting between the three kinds of `Operator`s
+
+- Worker-owned and Directed Cooperative, Mutual-Benefit, or Mutual Aid Organizations
+- Individual People
+- Legacy Organizations, such as traditional Non-profits, Governments, or Corporations
+
+## Social Responsibilities
+
+- Praxis
+  - Anti-(Fascist, Colonial, Imperial, White Supremacist, Patriarchal, Ableist, Heteronormative)
+    - Self and Community Love (see hooks, Lorde, Morrison, amb)
+    - Prison and Police Abolition (See Davis, Lorde, hooks)
+    - Self Determination (See Friere, hooks, Angelou, Davis)
+    - Intersectionality (See hooks)
+    - Harm Reduction
+    - Trauma Informed
+  - Universal access to healthy, reliable and secure sustenance, housing, healthcare and education.
+  - Mutual Aid
+  - Mutual Benefit
+
+- Regulatory Compliance and Legal
+  - Privacy Policies, Terms of Service, Content Policies
+  - Other Areas to be Aware of:
+    - Accessibility Regulation - https://www.w3.org/WAI/policies/
+    - Privacy Regulation
+      - https://en.wikipedia.org/wiki/California_Consumer_Privacy_Act
+      - https://en.wikipedia.org/wiki/Children%27s_Online_Privacy_Protection_Act
+      - https://en.wikipedia.org/wiki/General_Data_Protection_Regulation
+      - https://en.wikipedia.org/wiki/Do_Not_Track_legislation
+    - Illegal Content - https://en.wikipedia.org/wiki/Digital_Services_Act
+    - Monopoly Busting - https://en.wikipedia.org/wiki/Digital_Markets_Act
+
+### As a Cooperative
+
+### As a Person
+
+### As a Legacy Organization
+
+## Economic Model
```

Comment:
I assume this section would explain how to use convene in a cost-effective way by providing finance guidelines, legal FYI, etc.